src/axolotl/cli/config.py

```diff
@@ -159,6 +159,9 @@ def plugin_set_cfg(cfg: DictDefault):
     if cfg.get("plugins"):
         plugin_manager = PluginManager.get_instance()
         plugin_manager.cfg = cfg
+        # now that we have the finalized cfg, register the plugins individually
+        for plugin in plugin_manager.plugins.values():
+            plugin.register(cfg)
```

Comment:
We already do this [here](https://github.com/axolotl-ai-cloud/axolotl/blob/22810c97b72a397d877e27fe9b75b3be10920e27/src/axolotl/cli/config.py#L145)